src/Facebook/InstantArticles/Transformer/Rules/ParagraphFooterRule.php

```diff
@@ -8,36 +8,37 @@
  */
 namespace Facebook\InstantArticles\Transformer\Rules;
 
+use Facebook\InstantArticles\Elements\Element;
 use Facebook\InstantArticles\Elements\Paragraph;
 use Facebook\InstantArticles\Elements\Footer;
+use Facebook\InstantArticles\Validators\Type;
+use Facebook\InstantArticles\Transformer\Transformer;
 
 class ParagraphFooterRule extends ConfigurationSelectorRule
```

Comment:
We can probably deprecate this rule as well if we add Footer as a possible context for Paragraph.

Reply:
Indeed: #286